tracing-core/src/lib.rs

```diff
@@ -219,6 +216,14 @@ pub mod span;
 pub(crate) mod stdlib;
 pub mod subscriber;
 
+// Vendored version of spin 0.5.2 (0387621)
+// `mutex` and `once` modules only
+#[cfg(not(feature = "std"))]
+pub mod spin;
+
+// Vendored version of lazy_static 1.4.0 (4216696)
+pub mod lazy_static;
```

Comment:
Can we change this so that we only depend on a vendored `lazy_static` when the `std` feature is disabled, the way we do for `spin`? I'd prefer to use the a crates.io dependency whenever it's possible...

Reply:
Also, I don't think this needs to be `pub` — we only use the `lazy_static` macro internally, not in code generated by the macros. Can we make this private?
```suggestion
pub(crate) mod lazy_static;
```